ci-operator/config/opendatahub-io/kubeflow/opendatahub-io-kubeflow-e2e-commands.sh

```diff
@@ -0,0 +1,21 @@
+        #Gather tag and image for deployment
+        IN=$ODH_NOTEBOOK_CONTROLLER_IMAGE
+        arrIN=(${IN//:/ })
+        export IMG=${arrIN[1]}  
```

Comment:
@maroroman afaik, bash arrays are zero based so this would assign the imagetag name to `IMG` and `TAG` would be the empty string